.github/workflows/release-scheduler.yml

```diff
@@ -19,9 +19,12 @@ jobs:
       issues: write
     steps:
       - name: Harden Runner
-        uses: step-security/harden-runner@f086349bfa2bd1361f7909c78558e816508cdc10 # v2.8.0
+        uses: step-security/harden-runner@17d0e2bd7d51742c71671bd19fa12bdc9d40a3d6 # v2.8.1
         with:
-          egress-policy: audit
+          disable-sudo: true
+          egress-policy: block
```

Comment:
should we have the same change for other pipelines also? There seem to be many other yamls in our CICD that are using this action.

Reply:
Can only switch to block after it's run enough times to have data, and this is the only one with enough runs so far to switch.
But harden-runner only collects data on ubuntu workflows at the moment, so other yamls that use windows don't have data.
We could remove harden-runner from the windows workflows but it's more work to remove them and step-security would want them added back in, where I think it's not harmful to keep.